### ISSUE TITLE
default values in of in "Spacecraft Retrieval example

### ISSUE THREAD

**Issue by mjwolff** [kconnour/pyRT_DISORT#28]
### Feature request

TEMIS = 0 should be the default.  This is setting the emissivity of the "upper boundary condition" (analogous to the surface), which is not something that it done in terrestrial atmosphere RT.  So you want a value that is not going to create problems if someone turns on thermal emission and is not thinking to check this variable.

H_LYR variable name should be changed when you are setting the atmospheric scale height for vertical profiles.  I suggest something like H or even "H_scale".  

default value of H_LYR (which has dimension N+1 if there are N layers in the atmosphere) would just be the latitudes (AGL) of the layer boundaries (a.k.a. the z grid).  there are only needed by DISORT when using the pseudo-spherical approximation.  in the nomenclature of this example, right before the call to distort you could have a definition like so:  H_LYR = altitude_grid

### Priority

mainly a documentation request, but just in case some default values are set outside of the spacecraft retrieval example.

**Comment by mjwolff** [comment]
Yes, altitudes (oops).  I have been assuming same units.  Was km the test problem.

Should probably change variable to PLANET_RADIUS


------
Sent from an iPhone

On Apr 3, 2024, at 14:40, kconnour ***@***.***> wrote:

﻿

Fixing TEMIS is easy to fix and done.

H_LYR (which has dimension N+1 if there are N layers in the atmosphere) would just be the latitudes (AGL) of the layer boundaries...

should "latitudes" be "altitudes"? If so, are the units the same as EARTH_RADIUS?

H_LYR variable name should be changed when you are setting the atmospheric scale height for vertical profiles. I suggest something like H or even "H_scale".

I'm not sure how to interpret this. It seems like I should simply set H_LYR = z_grid and not have any calls to the scale height function. If H_LYR doesn't necessarily have any relation to the scale height, it's not clear to me that the scale_height function is particularly useful to include (but it 's not hurting anything, so I can certainly keep it around).

**Comment by kconnour** [comment]
Ok that'll be fixed momentarily. While I agree that EARTH_RADIUS sucks as a variable name, I'm sticking to using the DISORT convention and that's what they call it...

**Comment by mjwolff** [comment]
You mean because of f2py?  You mean that you can’t override a one-to-one match with the variable name?

**Comment by kconnour** [comment]
It's not due to f2py. I'm choosing the convention to use what DISORT asks for. I figure it'd be nicer to anyone who may have DISORT experience to leave the variable names the same within this package. I could modify f2py to make it rename EARTH_RADIUS into PLANET_RADIUS if desired, but I figure that might throw off someone (potentially).

**Comment by mjwolff** [comment]
It doesn’t make a difference to me, but I suspect it is more confusing to a new users.  However, if we create an example with pseudo-spherical geometry somewhere, then that will solve the problem.  I will work on one.
